students/GKim/lesson02/grid_printer.py

```diff
@@ -0,0 +1,48 @@
+"""
+FYI not my code , got help from you tube... but I understand and like this method.
+"""
+
+def print_row(c, a =3): 
+    """
+    function to print + and - using a 
+    for loop .  Using input from user the range is -1 because
+    from the first intial print the  section of the column is already created
+    this all prints accross
+    """
+    for _ in range(a-1):
+        print("+", end = "")
+        print(" - " * ((c-1)//2), end = "")
+    print("+")
+
+def print_column(c, a =3):
+    """
+    prints the "columns" " | " in the grid. just like the row first section is 
+    already created by the input given from user.  The colums print across 
+    """
+    for _ in range(a-1):
+        print("|", end = "")
+        print("   " * ((c-1)//2), end = "")
+    print("|")
+    
+
+def my_grid(c,a =3):
```

Comment:
This looks to be printing properly when given 1 parameter, ex: my_grid(5), my_grid(7). 

It doesn't properly handle the case when we pass 2 parameters though. The first parameter determines the nxn grid size and the second parameter determines the size of each cell. So for example, print_grid(5,3) should return a 5x5 grid where each cell is 3x3 (3 '-' across and 3 '|' long )